doc/developer-guide/api/functions/TSHttpSsnIdGet.en.rst

```diff
@@ -0,0 +1,44 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+.. default-domain:: c
+
+TSHttpSsnIdGet
+*******************
+
+Returns the unique identifier for client session.
+
+Synopsis
+========
+
+`#include <ts/ts.h>`
+
+.. function:: int TSHttpSsnIdGet(TSHttpSsn ssnp)
+
+Description
+===========
+
+:func:`TSHttpSsnIdGet` returns the unique identifier for this client session.
+
+Return Values
+=============
+
+Returns a :type:`int` as the unique ID
```

Comment:
Why is this `int` if the declared return type is `int64_t`?

Reply:
oops